app/src/main/java/org/schabi/newpipe/player/event/PlayerGestureListener.java

```diff
@@ -271,7 +271,10 @@ private boolean onScrollInMain(final MotionEvent initialEvent, final MotionEvent
             bar.setProgress((int) (bar.getMax() * Math.max(0, Math.min(1, oldBrightness))));
```

Comment:
After i rebased it, when you scroll all the way down and continue downwards the wheel jumps around a bit, while the brightness stays at 1%. My guess is that because here a float gets converted to an int. On my phone `bar.getMax()` returns 810, so with that conversion you lose something (810 * 0.01 = 8.1 => 8)

Changing it to `Math.max(0.01 , Math....)` doesn't help.